packages/zeplin-extension-style-kit/elements/layer.js

```diff
@@ -244,6 +248,18 @@ class Layer {
             new Height(new Length(layer.rect.height, { useRemUnit: useRemUnitForMeasurement }))
         ];
 
+        const bound = Bound.layerToBound(layer);
```

Comment:
We can do this computation only if `this.params.showPaddingMargin` is `true`, otherwise we would do it for nothing, right?